types/react-native/index.d.ts

```diff
@@ -501,6 +507,18 @@ export interface PressableProps extends AccessibilityProps, Omit<ViewProps, 'sty
      */
     onLongPress?: null | ((event: GestureResponderEvent) => void);
 
+    /**
+     * Called after the element loses focus.
+     * @platform windows
+     */
+    onBlur?: null | ((event: NativeSyntheticEvent<TargetedEvent>) => void);
+
+    /**
+     * Called after the element is focused.
+     * @platform windows
+     */
+    onFocus?: null | ((event: NativeSyntheticEvent<TargetedEvent>) => void);
+
```

Comment:
Wanted to call these two out, since they're a bit different than the norm. These currently exist only on Windows, with a PR to add to Android. tv-os was looking at doing something similar, along with some other platforms, so the thought is having an official typing would act as a contract for such platforms.